core/file/atime_spec.rb

```diff
@@ -16,13 +16,11 @@
   end
 
   platform_is :linux do
+    ## NOTE also that some Linux systems disable atime (e.g. via mount params) for better filesystem speed.
     it "returns the last access time for the named file with microseconds" do
-      3.times do
-        @atime = File.atime(@file)
-        break if @atime.usec > 0
-        sleep 0.001
-      end
-      @atime.usec.should > 0
+      require 'fileutils'
+      FileUtils.touch @file, mtime: Time.at(Time.now.to_i + 0.123456)
+      File.atime(@file).usec.should == 123456
```

Comment:
Indeed, this is a much better way to do it!
Could you use `File.utime` though?
No standard library should be required to run core specs.